packages/docusaurus-theme-classic/src/theme/NavbarItem/DefaultNavbarItem.tsx

```diff
@@ -61,6 +63,22 @@ function NavLink({
 }
 
 function NavItemDesktop({items, position, className, ...props}) {
+  const dropDownRef = React.useRef<HTMLDivElement>(null);
+  const dropDownMenuRef = React.useRef<any>(null); // TODO should find better solution for this. anything else will get a error when retrieve children.
```

Comment:
Doesn't something like `<UlHtmlElement | null>` works?

Reply:
I tested lots of solutions! Anything will get this error on focus method of childNodes!
![image](https://user-images.githubusercontent.com/24778140/89707933-c52b8f00-d987-11ea-8d04-176810a85541.png)

I can add <HtmlUlistElement | any>. is it okey?